packages/uswds-core/src/styles/mixins/helpers/checkbox-and-radio-colors.scss

```diff
@@ -131,8 +142,28 @@
           }
         }
       }
+      &:indeterminate,
+      &[data-indeterminate] {
+        &,
+        &:checked {
```

Comment:
@mahoneycm Curious about this line. Is the intention to have indeterminate styles take precedent when both indeterminate and checked attributes are present?

Reply:
Precisely! This was a part of the original contribution but it allows consistent display of the indeterminate styles for checked or unchecked checkboxes.

Reply:
After reading more about the spec for indeterminate state, my understanding is that a checkbox should be either `checked` OR `indeterminate`, not both. Indeterminate implies that it is _not_ checked, so I vote that we remove this explicit state of `:indeterminate:checked` from the Sass. 

Note that indeterminate styles will still take precedence over checked styles due to the order of the CSS, but I don't expect this to cause issues because the checkbox should have only one of these attributes. 

> ...radio and checkbox elements can be toggled between checked and unchecked states, but are sometimes in an indeterminate state, _neither checked nor unchecked_[^1]

[^1]: https://drafts.csswg.org/selectors/#indeterminate

Reply:
This sounds good to me! I'll remove and check for regressions 👍

Reply:
Resolved in [71c1cb9](https://github.com/uswds/uswds/pull/5713/commits/71c1cb982c8ea3941e3c6a7ba8ce1dc5a31195b9)